### PR TITLE
Document cache example for docker

### LINKED ISSUE #9688: Documentation for enabling atime when using minio in a Docker container
In the [Disk Caching Design](https://github.com/minio/minio/blob/master/docs/disk-caching/DESIGN.md) document it is mentioned that "the cache drives are required to be a filesystem mount point with atime support". But in the [Docker Quickstart Guide](https://docs.min.io/docs/minio-docker-quickstart-guide.html) there is no mention of enabling atime.

Some googling for how to do this did not turn up any clear answers. For example, [this stackoverflow question](https://stackoverflow.com/questions/57048660/how-do-i-create-a-docker-container-with-strictatime-set-on-the-root-filesystem) has no answers.

Some additional documentation on the best way to set up a cache volume in Docker would be very helpful.

### PR DESCRIPTION
**Pull request by poornas**
Fixes: #9688

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation needed
- [ ] Unit tests needed
- [ ] Functional tests needed (If yes, add [mint](https://github.com/minio/mint) PR # here: )

### CODE REVIEW

File: docs/disk-caching/DESIGN.md [hunk 1]
```diff
@@ -32,6 +32,20 @@ minio gateway <name> -h
      $ minio gateway s3
 ```
 
+### Run MinIO gateway with cache on Docker Container
+### Stable
+Cache drives need to have `strictatime` or `relatime` enabled for disk caching feature. In this example,mount xfs file system on /mnt/cache with `strictatime` or `relatime` enabled.
```
Comment (eco-minio) [inline docs/disk-caching/DESIGN.md]:
```suggestion
Cache drives need to have `strictatime` or `relatime` enabled for disk caching feature. In this example, mount the xfs file system on /mnt/cache with `strictatime` or `relatime` enabled.
```

**Review by eco-minio — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#7f288292-845f-4cd9-964b-b00aeec97f18) |
| mint-gateway-nas.sh | :x: [more...](#4ca0ed5a-790c-4c26-8590-dacac7e1afce) |
<a name="7f288292-845f-4cd9-964b-b00aeec97f18"></a>
### 9712-f317d85/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev7.minio.io:30353
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 8381387954b4:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 3 minutes and 17 seconds
(4/15) Running aws-sdk-ruby tests ... done in 15 seconds
(5/15) Running awscli tests ... done in 2 minutes and 45 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 2 minutes and 56 seconds
(8/15) Running minio-dotnet tests ... done in 2 minutes and 28 seconds
(9/15) Running minio-go tests ... done in 4 minutes and 3 seconds
(10/15) Running minio-java tests ... FAILED in 25 minutes and 42 seconds
{
  "name": "minio-java",
  "function": "listIncompleteUploads(String bucketName, String prefix)",
  "duration": 1643,
  "status": "FAIL",
  "error": "java.io.EOFException >>> [okio.RealBufferedSink.write(RealBufferedSink.java:115), io.minio.HttpRequestBody.writeTo(HttpRequestBody.java:78), okhttp3.internal.http.CallServerInterceptor.intercept(CallServerInterceptor.java:83), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:45), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121), okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:94), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121), okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147), okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121), okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:264), okhttp3.RealCall.execute(RealCall.java:93), io.minio.MinioClient.execute(MinioClient.java:1097), io.minio.MinioClient.execute(MinioClient.java:1236), io.minio.MinioClient.executePut(MinioClient.java:1410), io.minio.MinioClient.executePut(MinioClient.java:1424), io.minio.MinioClient.uploadPart(MinioClient.java:5492), io.minio.MinioClient.putObject(MinioClient.java:3740), io.minio.MinioClient.putObject(MinioClient.java:3870), FunctionalTest.listIncompleteUploads_test2(FunctionalTest.java:1790), FunctionalTest.runTests(FunctionalTest.java:4052), FunctionalTest.main(FunctionalTest.java:4192)]"
}
(10/15) Running minio-js tests ... done in 2 minutes and 30 seconds
(11/15) Running minio-py tests ... done in 9 minutes and 55 seconds
(12/15) Running s3cmd tests ... done in 1 minutes and 34 seconds
(13/15) Running s3select tests ... done in 1 minutes and 0 seconds
(14/15) Running security tests ... done in 0 seconds

Executed 14 out of 15 tests successfully.
```
<a name="4ca0ed5a-790c-4c26-8590-dacac7e1afce"></a>
### 9712-f317d85/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev6.minio.io:32441
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 6141e9de09e4:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 1 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 1 seconds
(4/15) Running aws-sdk-ruby tests ... done in 2 seconds
(5/15) Running awscli tests ... done in 2 minutes and 5 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 29 seconds
(8/15) Running minio-dotnet tests ... done in 32 seconds
(9/15) Running minio-go tests ... done in 54 seconds
(10/15) Running minio-java tests ... done in 28 seconds
(11/15) Running minio-js tests ... FAILED in 14 seconds
{
  "name": "minio-js",
  "function": "getBucketPolicy(bucketName, cb)",
  "args": "bucketName:minio-js-test-596c16ed-b3aa-4e19-a868-4841b6c233af",
  "duration": 12,
  "status": "FAIL",
  "error": "S3Error: The bucket policy does not exist at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:86:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:156:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:80:10) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (_stream_readable.js:673:10) at endReadableNT (_stream_readable.js:1201:12) at processTicksAndRejections (internal/process/task_queues.js:84:21)"
}
(11/15) Running minio-py tests ... done in 56 seconds
(12/15) Running s3cmd tests ... done in 17 seconds
(13/15) Running s3select tests ... done in 2 seconds
(14/15) Running security tests ... done in 0 seconds

Executed 14 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#9846eb45-2249-416a-91c1-942dd990fcdf) |
| mint-gateway-nas.sh | :x: [more...](#d139e3a0-0e43-4607-b0fa-00e274faff6a) |
<a name="9846eb45-2249-416a-91c1-942dd990fcdf"></a>
### 9712-fba3715/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev7.minio.io:31595
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 9b6954739350:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 7 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 3 minutes and 17 seconds
(4/15) Running aws-sdk-ruby tests ... done in 15 seconds
(5/15) Running awscli tests ... done in 3 minutes and 46 seconds
(6/15) Running healthcheck tests ... done in 1 seconds
(7/15) Running mc tests ... done in 3 minutes and 20 seconds
(8/15) Running minio-dotnet tests ... done in 1 minutes and 39 seconds
(9/15) Running minio-go tests ... done in 3 minutes and 47 seconds
(10/15) Running minio-java tests ... done in 27 minutes and 24 seconds
(11/15) Running minio-js tests ... FAILED in 1 minutes and 51 seconds
{
  "name": "minio-js",
  "function": "fPutObject(bucketName, objectName, filePath, metaData)",
  "args": "bucketName:minio-js-test-b056e940-2ae3-4b2e-8b00-f0771f72c8fe, objectName:datafile-65-MB, filePath:/tmp/datafile-65-MB",
  "duration": 967,
  "status": "FAIL",
  "error": "Error: read ECONNRESET at TCP.onStreamRead (internal/stream_base_commons.js:205:27)"
}
(11/15) Running minio-py tests ... done in 9 minutes and 58 seconds
(12/15) Running s3cmd tests ... done in 1 minutes and 25 seconds
(13/15) Running s3select tests ... done in 59 seconds
(14/15) Running security tests ... done in 0 seconds

Executed 14 out of 15 tests successfully.
```
<a name="d139e3a0-0e43-4607-b0fa-00e274faff6a"></a>
### 9712-fba3715/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev6.minio.io:31280
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 88f4cf33d48d:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 0 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 1 seconds
(4/15) Running aws-sdk-ruby tests ... done in 2 seconds
(5/15) Running awscli tests ... done in 2 minutes and 5 seconds
(6/15) Running healthcheck tests ... done in 1 seconds
(7/15) Running mc tests ... done in 32 seconds
(8/15) Running minio-dotnet tests ... done in 27 seconds
(9/15) Running minio-go tests ... done in 40 seconds
(10/15) Running minio-java tests ... done in 31 seconds
(11/15) Running minio-js tests ... FAILED in 9 seconds
{
  "name": "minio-js",
  "function": "getBucketPolicy(bucketName, cb)",
  "args": "bucketName:minio-js-test-f503012e-0947-4760-a33f-e04bac941b0e",
  "duration": 7,
  "status": "FAIL",
  "error": "S3Error: The bucket policy does not exist at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:86:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:156:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:80:10) at DestroyableTransform.prefinish (node_modules/readable-stream/lib/_stream_transform.js:129:10) at prefinish (node_modules/readable-stream/lib/_stream_writable.js:611:14) at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:620:5) at endWritable (node_modules/readable-stream/lib/_stream_writable.js:643:3) at DestroyableTransform.Writable.end (node_modules/readable-stream/lib/_stream_writable.js:571:22) at IncomingMessage.onend (_stream_readable.js:673:10) at endReadableNT (_stream_readable.js:1201:12) at processTicksAndRejections (internal/process/task_queues.js:84:21)"
}
(11/15) Running minio-py tests ... done in 57 seconds
(12/15) Running s3cmd tests ... done in 17 seconds
(13/15) Running s3select tests ... done in 2 seconds
(14/15) Running security tests ... done in 0 seconds

Executed 14 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by BigUstad — APPROVED** [review]
LGTM

**Review by eco-minio — APPROVED** [review]
LGTM

**Pull request merged**